File: Homeworks/JSFundamentals/Fucntions/appearanceCount.js
var array = [2, 3, 2, 2, 2, 2, 2, 2, 3, 4, 4, 5, 6, 7, 8, 9],
    givenNumber = 2;

function countNumberInArr(array, number) {
    var count = 0;

    for (var i = 0; i < array.length; i++) {
        if (array[i] == number) {
            count += 1;
        }
    }
    return count;
}

console.log(countNumberInArr(array, givenNumber));

File: Homeworks/C#2/Methods/04.AppearanceCount/AppearanceCount.cs
using System;
using System.Linq;

//Problem 4. Appearance count

//Write a method that counts how many times given number appears in given array.
//Write a test program to check if the method is workings correctly.

class AppearanceCount
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        string textNumbers = Console.ReadLine();
        var numbers = textNumbers.Split(' ').ToArray();
        int[] array = new int[numbers.Length];
        for (int i = 0; i < numbers.Length; i++)
        {
            array[i] = int.Parse(numbers[i]);
        }
        //int[] array = { 2, 3, 4, 5, 6, 7, 7, 7, 7, 7, 7, 7, 9, 9, 12, 13, 9, 9 };
        int number = int.Parse(Console.ReadLine());
        CountElementsInArray(array, number);
    }

    static void CountElementsInArray(int[] array, int number)
    {
        int count = 0;
        for (int i = 0; i < array.Length - 1; i++)
        {
            if (number == array[i])
	        {
                count++;
	        }
        }
        Console.WriteLine(count);
    }
}

